doc/admin-guide/plugins/s3_auth.en.rst

```diff
@@ -36,23 +36,25 @@ Using the plugin in a remap rule would be e.g.::
 
    ...  @plugin=s3_auth.so @pparam=--access_key @pparam=my-key \
                            @pparam=--secret_key @pparam=my-secret \
+                           @pparam=--session_token @pparam=my-token \
                            @pparam=--virtual_host
 
 
 Alternatively, you can store the access key and secret in an external configuration file, and point the remap rule(s) to it::
 
    # remap.config
 
-   ...  @plugin=s3_auth.so @pparam=--config @pparam=s3_auth_v2.config
+   ...  @plugin=s3_auth.so @pparam=--config @pparam=s3_auth_v4.config
 
 
 Where ``s3.config`` could look like::
 
-    # s3_auth_v2.config
+    # s3_auth_v4.config
 
     access_key=my-key
     secret_key=my-secret
-    version=2
+    session_token=my-token
+    version=4
     virtual_host=yes
```

Comment:
`virtual_host` is a v2 specific parameter. We have 2 options here:

1. remove `virtual_host` from this example (otherwise plugin will warn in the log on each realod/restart)

1. change the example back to v2 like it was before (you have already added a good example in the v4 specific section) and remove `session_token=my-token`

Reply:
Thanks, opting for 2.